### DISCUSSION TITLE
Kafka 3.0

### DISCUSSION THREAD

**Discussion by ThePoipoi** [strimzi/strimzi-kafka-operator#4598] [General]
Hi, it's me again. Not for help, but just for a quick question. 
If I understand well, Strimzi is currently supporting Kafka 2.7 with Scala 2.12.
But Kafka is also using Scala 2.13 with the 2.7 version. And Apache is now working on the 3.0.0 version (still in snapshot).

Are you gonna migrate to Kafka 3.0.0, and at least, to Scala 2.13 ?

That's all :)
Thanks.

**Comment by scholzj** [comment]
We expect to support Kafka 3.0, although the details what it will or will not do are still a bit unclear at this point I think. We are currently using Scala 2.12 for all Kafka releases. But I do not think we have any hard dependency on it. So if you think 2.13 would have some advantages, feel free to let us know.

**Reply by ThePoipoi** [reply]
To be honest, it was only curiosity, I didn't check anything on Scala, just saw Apache support both version.
And yes, it's a bit unclear on what is going on for the 3.0.0
I just heard somewhere that they are trying to remove Zookeeper, but no more.

Thanks for your answer :)

**Reply by scholzj** [reply]
Yes, 3.0 should be the first version which can work without Zookeeper. But I think it is not clear if all the current features will be supported there, what will be the upgrade process (if upgrade would be even possible etc.). So we need to understand these.
